Misc/NEWS.d/next/Build/2024-12-20-09-03-22.gh-issue-128104.m_SoVx.rst

```diff
@@ -0,0 +1,2 @@
+Remove ``Py_STRFTIME_C99_SUPPORT`` and require C99-compliant ``strftime``
+datetime specifier support at build time when not cross-compiling.
```

Comment:
```suggestion
:mod:`datetime` specifier support at build time when not cross-compiling.
```

Reply:
Is this correct? I'm not referring to the Python datetime module here, just datetime specifiers in general.

Reply:
There are some nuances. See bottom of https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes and https://docs.python.org/3/library/datetime.html#technical-detail. I don't remember which nuance is the correct one for this specific test so carefully check that please!

Reply:
Will do, thanks for the references — I think your parenthetical reference is correct but I'll double check.

Reply:
Updated in https://github.com/python/cpython/pull/128106/commits/0a9e225960d94a67913d30edc22cf1b92ab47049